Parsers/pfSense/logstash-pfsense-geoIp.txt

```diff
@@ -0,0 +1,22 @@
+// pfSense | opnSense GeoIp Traffic Parser
+// Last Updated Date: March 09, 2021
+//
+// LogStash Instructions:
+// pfSense/opnSense - Logstash Configuration - https://github.com/noodlemctwoodle/pf-azure-sentinel
+//
+// Parser Notes:
+// This parser will parse pfSense or opnSense WAN logs and tag them with GeoIP data using MaxMind db or Logstash GeoIP db.
+//
+// Usage Instruction :
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. pf_GeoIPDestination) or
+// paste in the command into your Azure Sentinel logs and run the query.
+//
+//
```

Comment:
We need an event sample and I don't see any Sample data for pfSense - see step 4 from here - https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md#build-the-connector.

Reply:
@shainw Added screenshot

Reply:
@noodlemctwoodle - Sorry if not clear, we need an actual sample event in a string format as would show up in an export of the row of data into a CSV or the like, then we can test the parser.